Dockerfile

```diff
@@ -19,7 +19,7 @@ RUN apk update && apk upgrade && \
     ${BUILD_DEPENDENCIES} \
     ${APP_DEPENDENCIES} && \
     gem install bundler --no-ri --no-rdoc && \
-    cd ${DCAF_DIR} ; bundle install --without development test && \
+    cd ${DCAF_DIR} ; bundle install --without development && \
```

Comment:
since we're not deploying a docker image or anything, should just do a straight up bundle install here?